srcpkgs/wireshark/template

```diff
@@ -6,8 +6,9 @@ build_style=cmake
 hostmakedepends="pkg-config flex perl python3-devel mit-krb5-devel qt5-tools
  qt5-host-tools qt5-qmake"
 makedepends="libpcap-devel zlib-devel lua52-devel mit-krb5-devel gnutls-devel
- libmaxminddb-devel libgcrypt-devel libcap-devel sbc-devel c-ares-devel
- qt5-devel qt5-tools-devel qt5-multimedia-devel qt5-svg-devel"
+ libmaxminddb-devel libgcrypt-devel libcap-devel sbc-devel c-ares-devel libxml2-devel
+ libssh-devel libnl3-devel spandsp-devel tiff-devel libcap-progs liblz4-devel
```

Comment:
Is adding `tiff-devel` related to enable sshdump and ciscodump?

Reply:
it is needed by spandsp

/usr/include/spandsp.h:43:10: fatal error: tiffio.h: No such file or directory
 #include <tiffio.h>
          ^~~~~~~~~~
compilation terminated.

Reply:
ok